### DISCUSSION TITLE
`useBlockStatements`: corresponding option for like `multi-line` in eslint

### DISCUSSION THREAD

**Discussion by xnuk** [biomejs/biome#1845] [Rule suggestion]
Personally I like this rule from [standard.js](https://standardjs.com/rules):

> **For multi-line if statements**, use curly braces.
> eslint: [`curly`](https://eslint.org/docs/latest/rules/curly)
> ```javascript
> // ✓ ok
> if (options.quiet !== true) console.log('done')
> ```
> ```javascript
> // ✓ ok
> if (options.quiet !== true) {
>   console.log('done')
> }
> ```
> ```javascript
> // ✗ avoid
> if (options.quiet !== true)
>   console.log('done')
> ```

So I hope to use this in Biome.

\+ Deleting `useBlockStatements` and just put in the formatter is also okay.

**Comment by JustJoostNL** [comment]
@xnuk (or others) did you find any solution for this in Biome?
